sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1627,6 +1627,10 @@ def _add_argparse_args(cls, parser):
         action='append',
```

Comment:
Looks like this is still failing format/lint checks - could you follow https://cwiki.apache.org/confluence/display/BEAM/Python+Tips#PythonTips-LintandFormattingChecks to fix?

Reply:
Thank you for comments. I will check that. :)